srcpkgs/goattracker-stereo/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'goattracker-stereo'
+pkgname=goattracker-stereo
+version=2.76
+revision=1
+wrksrc=gt2stereo
+makedepends="SDL-devel"
+short_desc="C64 music tracker (stereo)"
+maintainer="Zeinok <zeinok@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://sourceforge.net/projects/goattracker2/"
+distfiles="${SOURCEFORGE_SITE}/goattracker2/GoatTracker%202%20Stereo/${version}/GoatTracker_${version}_Stereo.zip"
+checksum=878a4194ecf48c48a89ba0a07fcce6d1ad27b3b1c9ec86e52160960ed0199f89
+
+do_build() {
+	cd ${wrksrc}/trunk/src/
+	make clean
+	cd ${wrksrc}/trunk/src/bme/
+	make
+	cd ${wrksrc}/trunk/src/
+	make
+}
+
+do_install() {
+	vbin trunk/linux/gt2stereo
+	vmkdir usr/lib/gt2stereo
+	vinstall trunk/linux/ins2snd2 755 usr/lib/gt2stereo
+	vinstall trunk/linux/mod2sng2 755 usr/lib/gt2stereo
+	vinstall trunk/linux/ss2stereo 755 usr/lib/gt2stereo
+	vinstall ${FILESDIR}/gt2stereo.desktop 644 usr/share/applications
+	vinstall ${FILESDIR}/gt2stereo.png 644 usr/share/pixmaps
```

Comment:
Why do we need to vendor them?

Reply:
For Debian package it has dekstop file included so I just kind of followed them, and I'm not sure about not providing desktop files since it's a GUI program

Reply:
NixOS also have desktop file for it fyi  
https://github.com/NixOS/nixpkgs/blob/0794fb0419cb0bc1622905b281e0699b9c2f9e3c/pkgs/applications/audio/goattracker/default.nix#L13

Reply:
Please try talking to upstream about including a desktop file, it doesn't make sense for every distro to carry a random version of it themselves.

Reply:
thanks for explaining the situation, then I think not supplying desktop files would be better for now.